engines/python/setup/djl_python/rolling_batch/rolling_batch.py

```diff
@@ -407,19 +407,14 @@ def try_catch_handling(self, *args, **kwargs):
             return func(self, *args, **kwargs)
         except Exception:
             logging.exception("Rolling batch inference error")
-            err = {
-                "data": "",
-                "last": True,
-                "step_token_num": 0,
-                "code": 424,
-                "error": ""
-            }
-            results = []
-            for i in range(
-                    len(self.active_requests) + len(self.pending_requests)):
-                results.append(err)
+            for request in self.active_requests:
+                token = Token([-1], "", -1, None)
+                request.set_next_token(token,
```

Comment:
Do you think we should also set error response code? https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_runtime_InvokeEndpointWithResponseStream.html#sagemaker-runtime_InvokeEndpointWithResponseStream-response-ModelStreamError